themes/soe_basic/src/scss/components/stat_card/stat_card.scss

```diff
@@ -0,0 +1,103 @@
+@charset 'UTF-8';
+@import 'decanter/core/src/scss/decanter-no-markup';
+@import 'decanter/core/src/scss/components/card/index';
+
+@mixin stat-card-dark-bg {
+  .stat-button {
+    box-shadow: inset 0 0 0 2px $su-color-white;
+    color: white;
+
+    &:hover,
+    &:focus {
+      box-shadow: inset 0 0 0 2px $su-color-white;
+      color: $su-color-white;
+    }
+  }
+
+  a {
+    color: $su-color-white;
+
+    &::after {
+      background-color: $su-color-white;
+    }
+
+    &:hover,
+    &:focus {
+      &::after {
+        background-color: $su-color-white;
+      }
+    }
+  }
+}
+
+.su-stat-card {
+  .stat-card-headline {
+    h2,
+    h3,
+    h4,
+    h5 {
+      font-weight: 700;
+    }
+  }
+
+  .stat-card-stat {
+    font-weight: 700;
+  }
+}
+
+.stat-icon-d18100 {
+  .stat-card-icon {
+    color: #d18100;
```

Comment:
I'm actively troubleshooting why `$soe-color-[color name]` is throwing an error 🤔